systems/frontends/modules/haproxy_base/default.nix

```diff
@@ -4,23 +4,14 @@
   pkgs,
   ...
 }: {
-  networking.firewall.allowedTCPPorts = [
-    # byond
-    6001
-    20002
-    # Game Servers
-    8989 # funnyname
-  ];
   systemd.tmpfiles.rules = [
     "d /var/lib/haproxy 770 ${config.services.haproxy.user} ${config.services.haproxy.group}"
   ];
   services.haproxy = {
     enable = true;
     config =
-      "# ==== GLOBAL CONFIG ====\n" +
-      builtins.readFile ../../../haproxy_relay_global.conf +
-      "\n\n# ==== LOCAL CONFIG ====\n" +
-      builtins.readFile ./haproxy.conf;
+      "# ==== GLOBAL CONFIG ====\n"
+      + builtins.readFile ./haproxy.conf;
```

Comment:
Has this been tested to work properly?

Reply:
Negative, I do not have a setup yet. Tbh I was hoping for this to get tested on staging so that i wouldn't have to worry about doing it myself